addons/html_editor/static/src/main/media/image_plugin.js

```diff
@@ -349,20 +364,21 @@ export class ImagePlugin extends Plugin {
         }
     }
 
-    openImageTransformation(image) {
-        if (registry.category("main_components").contains("ImageTransformation")) {
+    handleImageTransformation(image) {
+        if (this.isImageTransformationOpen()) {
+            this.closeImageTransformation();
             return;
         }
-        Promise.resolve().then(() => {
-            this.document.getSelection()?.removeAllRanges();
-        });
```

Comment:
Did you have a look as to why this line was added? Because it's kind of a convoluted issue that prompted me to create a "refactor the toolbar" task when I tried to remove it to do it in another way and could not figure out an easy/proper way to do it differently. So if you didn't break the case that this line was meant to handle then I'd be interesting in knowing how you did it... ^^'